### PR TITLE
Improve search experience

### PR DESCRIPTION
**Pull request by DengYiping**
Final release.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -2,13 +2,14 @@
 import json
 import requests
 import telebot
-
+from telebot import types
 LOKLAK_API_URL = "http://loklak.org/api/search.json?q={query}"
-
-bot = telebot.TeleBot(os.environ['LOKLAK_TELEGRAM_TOKEN'])
+#put your token in the line below
+bot = telebot.TeleBot("PUT YOUR TOKEN HERE")
```
Comment (sudheesh001) [inline bot.py]: refactor this into a new file called `token.py` from which do

`from token import TOKEN` where
`TOKEN = '****Your long token over here***'`

**Comment by sevazhidkov** [comment]
Thank you, cool search method. Why did you change the method of getting token? Defining tokens in code isn't good experience.

**Comment by DengYiping** [comment]
I changed it to what @sudheesh001 suggested. Now you can safely merge. I also add some new function like doing /user:CBS

File: token.py [hunk 2]
```diff
@@ -0,0 +1 @@
+TOKEK = "YOUR TOKEN CODE IS HERE"
```
Comment (sudheesh001) [inline token.py]: Nit: `TOKEN`

Comment (DengYiping) [inline token.py]: Thanks

**Comment by DengYiping** [comment]
@sudheesh001 @sevazhidkov When I am trying to fix the compatibility with Python 3, I find out that one should not have a file named token.py in the dictionary because it is a system module. It only cause trouble in Python 3 on my laptop. However, we should not do that. Now the code is working under Python 3.

**Comment by sevazhidkov** [comment]
@sudheesh001 Do you think that defining token in code is a good idea?

**Comment by DengYiping** [comment]
@sevazhidkov You can safely merge right now. There is no token.py.

**Comment by sudheesh001** [comment]
@sevazhidkov Looks like @DengYiping has refactored the code, there's no more a definition of token in the code.

**Comment by sevazhidkov** [comment]
@sudheesh001 I think that token should take from os environ variable.

**Comment by sevazhidkov** [comment]
But I can fix it myself, it's not problem

**Comment by sevazhidkov** [comment]
For example, if someone wants to host bot's source code in Github public repo, but also he wants to deploy bot in Heroku (or other cloud hosting), so he can just set env variable in dashboard of cloud hosting

**Comment by sevazhidkov** [comment]
@DengYiping Thank you, very good work

**Comment by DengYiping** [comment]
Not a problem. Great job.

**Pull request merged**